Assets/Dev/Si Wai/PermanentPlayerStatistic.cs

```diff
@@ -0,0 +1,106 @@
+
+using UnityEngine;
+[System.Serializable]
+public class PermanentPlayerStatistic : SaveSystem
+{
+    public PermanentStatistic Speed = new();
+    public PermanentStatistic JumpForce = new();
+    public PermanentStatistic MaxHealth = new();
+    private float health;
+    public float Health {
+        get => health;
+        set => health = value > 0 ? value : 0;
```

Comment:
should this be in here? it's not permament like maxHealth

Reply:
nope, should not be here

Reply:
permanentStats should not have absolute values. Permanent stats should ONLY be a list of modifiers.
and the current stats (Aka the stats the game actually uses to calculate stuff). Those stats should just copy over these permanent stats. 
That way whenever you start a run, or you start up the game. you always have the modifiers of the permanent stats and it will not change the functionality of the rest of the game